pkg/cli/cli.go

```diff
@@ -39,13 +40,20 @@ func Init(searchPath string, excludeDirs []string) CLI {
 
 	fileTypes := getFileTypes()
 
-	stdoutReporter := reporter.StdoutReporter{}
+	var reporterType reporter.Reporter
+
+	if reportType == "json" {
+		reporterType = reporter.JsonReporter{}
+	} else {
```

Comment:
Might want to throw an error if the report type is not json or standard just so the user is not confused.

Reply:
I'll add it in validator.go::getFlags where it checks the searchPathPtr is not equal to an empty string. Means I don't have to change the cli.go::Init function to return an error.